tests/streamlit_ui_tests/features/column_control_panel.spec.ts

```diff
@@ -66,7 +62,7 @@ test.describe('Column Control Panel', () => {
         
         await mito.getByText('Add Filter').click();
         await mito.getByText('Add a Group').click();
-        await mito.locator('.spacing-row', { hasText: 'Where' }).locator('input').fill('7');
+        await fillInput(mito, 'Where', '7');
```

Comment:
Want to update the code config, import tests to use this too?

Reply:
I will when it's merged :)